lib/temporal/client.rb

```diff
@@ -9,15 +9,50 @@
 
 module Temporal
   class Client
+    # @return [String] Namespace used for client calls.
     attr_reader :namespace
 
-    # TODO: More argument to follow for converters, codecs, etc
+    # Create a Temporal client from a service client.
```

Comment:
The term `service client` isn't defined anywhere AFAICT.

Reply:
Yep, will update 👍